bin/lw_agent

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/perl -T
```

Comment:
nit pick: not portable
`
#!/usr/bin/env perl
`

Reply:
Changing